fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -1799,7 +1799,7 @@ private Partition createPartitionWithIndices(String clusterName, long dbId, long
             }
 
             if (!ok || !countDownLatch.getStatus().ok()) {
-                errMsg = "Failed to create partition[" + partitionName + "]. Timeout:" + (timeout / 1000) + " seconds.";
+                errMsg = "The BE is not available,please check BE status!";
```

Comment:
How shall we figure out that the failure is due to really timeout or BE down by the hint msg then? Maybe we should check out BE status here and make the hint clearer.

Reply:
Yes I agree with you that we should give users a more explicit be status.

Reply:
> > if some BEs failed, but the majority of them success, can we treat the create as successful ?
> 
> So the conclusion is no, right?

fe will report regularly according to the full amount of tasks of be, and decide which tasks need to be resent and which tasks need to be discarded, So the conclusion is no.

Reply:
> > > if some BEs failed, but the majority of them success, can we treat the create as successful ?
> > 
> > 
> > So the conclusion is no, right?
> 
> fe will report regularly according to the full amount of tasks of be, and decide which tasks need to be resent and which tasks need to be discarded, So the conclusion is no.

So the result of executing the table is either success or failure, there will be no unknown results, am I right?

Reply:
> So the result of executing the table is either success or failure, there will be no unknown results,

@yujun777 may means that if the majority replicas for each tablet is created successfully, may we treat the creation as successful? Just like the criteria for loading.

 The current implementation does indicate that creation is successful when all replicas are created successfully, or it failed anyway. Maybe we could modify the implementation to achieve the goal if needed.

Reply:
It is also okay then if u find it hard or unnecessary to do this optimization. Then just make the hint msg clearer so that we could find out the creation failure is due to real timeout, or some BE error including BE crash.